README.md

```diff
@@ -4,13 +4,22 @@ Async Include
 
 A simple script to asynchronously include content on an HTML page with simple markup that “just works”.
 
-    <div data-async-url="/some-url/"></div>
+    <div data-async-include-url="/some-url/"></div>
+
+Just include the `async-include.js` file on your page, activate the replace when you want like when the document is ready:
```

Comment:
This sentence reads awkwardly.

> Just include the `async-include.js` file on your page, activate the replace when you want like when the document is ready:

maybe instead

> Add the `async-include.js` script to your page and then activate it by calling the `.asyncInclude()` jQuery method (perhaps when the document is ready):

IDK, mostly just the “…when you want like when the…” feels awkward.